lib/src/utils/constants.dart

```diff
@@ -34,7 +34,7 @@ extension ShowSnackBar on BuildContext {
   }) {
     showSnackBar(
       message,
-      backgroundColor: Theme.of(this).errorColor,
+      backgroundColor: Theme.of(this).colorScheme.error,
```

Comment:
suggestion: Maybe is more accurate to use colorscheme.errorContainer. 
Maybe is also worht passing the text color to the snackbar. In this case it can be colorscheme.onErrorContainer

Reply:
@Csierram96 
Thanks for the suggestion! I didn't know that colorScheme has a `errorContainer` property! It sounds like the perfect use case for this one. I will update the code using `colorScheme.errorContainer` and `colorScheme.onErrorContainer`!

Reply:
No problem at all, happy to help!. 
This website is really useful for material 3 stuff: https://flutter-experimental-m3-demo.web.app/#/

Reply:
Thanks for sharing. This is awesome! I will read it through!